### PR TITLE
Add Stack Overflow and all other Stack Exchange sites

### PR DESCRIPTION
**Pull request by verhovsky**
Homepage before

Screenshot 2023-09-20 at 11 59 08 AM

Homepage after

Screenshot 2023-09-20 at 12 00 39 PM

Question before

Screenshot 2023-09-20 at 11 59 10 AM

Question after

Screenshot 2023-09-20 at 12 00 56 PM

### CODE REVIEW

**Comment by mig4ng** [comment]
Hi @verhovsky , this is amazing work. I tried it in my computer and it seems to remove a lot of stuff.

However, the left sidebar or something, while being removed changes the size and all the content is moved to the left.

I tried to cleanup StackOverflow myself (just the questions) and I came up with this 3 simple lines:
```
! Hide StackOverflow left sidebar
stackoverflow.com##nav > .nav-links

! Hide StackOverflow right sidebar
stackoverflow.com##div > #sidebar

! Hide StackOverflow browse more bottom notice
stackoverflow.com##h2.bottom-notice > div
```

Do you think you can change the PR a bit to only change those for now? And incrementally add the other ones?
It is harder for me to understand and test what is happening everywhere all at once, even though everything is very well documented.

Feel free to change the values I set above to other ones.

**Comment by mig4ng** [comment]
Looking pretty good so far @verhovsky ! 🚀 

The before and after now in askubuntu and others with that enormous banner on top are amazing.

**Comment by mig4ng** [comment]
Tag me when you finish updating this so I give it a final review, thanks!

**Comment by verhovsky** [comment]
@mig4ng take a look now

File: stackoverflow.txt [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .user-details:style(margin-left:0px!important)
+! Users' points
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.post-signature > .user-info > .user-details > .-flair
+! Profile pictures on Top Questions page
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.s-post-summary--meta > .s-user-card > .s-user-card--avatar
+! Users' points next to username on Top Questions page
+stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.s-post-summary--meta > .s-user-card > .s-user-card--info > .s-user-card--awards
+! Hide annoying usernames
+! Put username text to match on (case insensitive) between the //
+!stackoverflow.com,stackexchange.com,askubuntu.com,stackapps.com,mathoverflow.net,superuser.com,serverfault.com##.s-user-card--link > .flex--item:has-text(//i)
```
Comment (mig4ng) [inline stackoverflow.txt:55]: What does the `:has-text(//i)` do?

Comment (mig4ng) [reply stackoverflow.txt:55]: @verhovsky

Comment (mig4ng) [reply stackoverflow.txt:55]: From what I understand it is a boilerplate to remove annoying users, that is not currently in use.

**Review by mig4ng — APPROVED** [review]

**Comment by mig4ng** [comment]
Thank you so much for your contribution @verhovsky ❤️ 

Can I tag you if in the future this needs update/maintenance?

File: stackoverflow.txt (hunk 1 above)
Comment (verhovsky) [reply stackoverflow.txt:55]: That's correct, it's just an example of how to do it.

**Comment by verhovsky** [comment]
Of course. Thanks for merging it, I hope other people use it.

**Pull request merged**